indico/htdocs/sass/modules/_event_management.scss

```diff
@@ -490,16 +490,60 @@ th.i-table {
 }
 
 #person-list {
-    .affiliation-column, .name-column {
-        max-width: 20em;
-    }
-
     .no-role {
         opacity: 0.25;
         .badge {
             background-color: $dark-gray;
         }
     }
+
+    thead {
+        display: block;
+        position: relative;
+        background-color: $light-gray;
+        @include border-bottom();
+    }
+
+    tbody {
+        display: block;
+        max-height: 50em;
+        overflow: auto;
+    }
+
+    .checkbox-column {
+        min-width: 2.5em;
+    }
+
+    .name-column, .affiliation-column {
+        max-width: 20em;
+        min-width: 20em;
```

Comment:
This is the same as adding `width: 20em;`.
Why do we need to specify the width of the columns?